doc/md/fundamentals/5-actors-async.md

```diff
@@ -124,6 +124,28 @@ Unlike a local function call, which waits for the result before continuing, a sh
 
 If you `await` the same future again, it just returns the same result or error. Even if the future is already done, `await` will briefly suspend all pending state changes and outgoing messages. This means that you can rely on every `await` to commit state, whether its future is still in progress or already completed.
 
+## Using `await?` to efficiently await concurrent futures
+
+An `await` will always suspend execution and commit state, even if its future is already complete.
+
+When several futures are issued in parallel and racing to complete, it can be more efficient to opt out of the unconditional behavior of `await`
+and immediately continue with a result when it is available:
```

Comment:
```suggestion
When several futures are issued in parallel and racing to complete, it can be more efficient to opt out of the unconditional behavior of `await` and immediately continue with a result when it is available:
```